docs/release-notes/rl-0.8.md

```diff
@@ -102,6 +102,7 @@
   - `mini.trailspace`
   - `mini.visits`
 - Add [fzf-lua](https://github.com/ibhagwan/fzf-lua) in `vim.fzf-lua`
+- Add options to define highlights under `vim.highlight`
```

Comment:
You should be able to refer to the option with a documentation link with `[](#opt-vim.highlight)` since a product of `mkOption`